modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/java/com/liferay/object/dynamic/data/mapping/form/field/type/internal/attachment/AttachmentDDMFormFieldTemplateContextContributor.java

```diff
@@ -82,6 +83,9 @@ public Map<String, Object> getParameters(
 			"fileSource", ddmFormField.getProperty("fileSource")
 		).put(
 			"maximumFileSize", maximumFileSize
+		).put(
+			"overallMaximumUploadRequestSize",
```

Comment:
Hey @danvb112, I'm kind of confused about this variable name. What this name exactly means?